### DISCUSSION TITLE
Clustering

### DISCUSSION THREAD

**Discussion by llalmall** [hnolCol/instantclue#39] [Q&A] [answered]
I am using instantclue (the latest ver.) on mac, I can't find the clustering mode under 'model' section in Analysis box. How can I add the clustering mode? There are only several things that can add the lines on the plots in the analysis box.

**Comment by hnolCol** [comment, marked as answer]
HI, do you mean the "clustering mode" for hierarchical clustering (e.g. heatmap?). If yes, please see the image below. You can set the clustering metric and method in the settings - they are automatically saved when you close the dialog. Please note that you have to replot the heatmap (e.g. click again on the icon) to make the change in the setting visible. 
![discussHierarchicaSetting](https://user-images.githubusercontent.com/24191910/152754652-688a43d5-accb-43f6-96bd-3d7f936c1e14.png)
